src/price-formatter.ts

```diff
@@ -0,0 +1,3 @@
+export function formatPrice(price: number): string {
+  return `R$${price.toFixed(2)}`;
+}
```

Comment:
boa! Acho que seria bom considerar que o price viria em centavos do servidor, ou seja, considere o `price` como 10000 para 100 reais, sendo assim, seria importante dividir por 100 antes de fazer o toFixed.

```suggestion
export function formatPrice(price: number): string {
  return `R$${(price / 100).toFixed(2)}`;
}
```